Provider/src/FirebirdSql.EntityFrameworkCore.Firebird/Scaffolding/Internal/FbDatabaseModelFactory.cs

```diff
@@ -192,11 +193,20 @@ ORDER BY
 
 		private void GetColumns(DbConnection connection, IReadOnlyList<DatabaseTable> tables, Func<string, string, bool> tableFilter)
 		{
+			var identity_type = "null";
+
+			if (connection.State == ConnectionState.Open)
+			{
+				var serverVersion = FbServerProperties.ParseServerVersion(connection.ServerVersion);
```

Comment:
I would put this into `DatabaseModel Create(DbConnection connection, DatabaseModelFactoryOptions options)` method and create a property for it.